File: simple-demo/javascript/simple-demo.js
/**
 * Kledo API Demo - JavaScript/Node.js (tanpa npm, Node 18+)
 * Jalankan: node simple-demo.js
 */

const API_HOST = 'http://xxx.api.kledo.com/api/v1';
const ACCESS_TOKEN = 'your_token_here';

const url = `${API_HOST}/finance/accounts?sort_by=name&order_by=asc&per_page=5`;
const res = await fetch(url, {
  headers: { 'Accept': 'application/json', 'Authorization': `Bearer ${ACCESS_TOKEN}` }
});
const data = await res.json();
console.log(JSON.stringify(data, null, 2));

File: simple-demo/python/simple-demo.py
"""
Kledo API Demo - Python (tanpa pip, pakai urllib bawaan)
Jalankan: python simple-demo.py
"""

import urllib.request
import json

API_HOST = 'http://xxx.api.kledo.com/api/v1'
ACCESS_TOKEN = 'your_token_here'

url = f'{API_HOST}/finance/accounts?sort_by=name&order_by=asc&per_page=5'
req = urllib.request.Request(url, headers={
    'Accept': 'application/json',
    'Authorization': f'Bearer {ACCESS_TOKEN}'
})
with urllib.request.urlopen(req) as r:
    data = json.load(r)
print(json.dumps(data, indent=2, ensure_ascii=False))
